spec/unit/util/execution_spec.rb

```diff
@@ -144,6 +155,20 @@ def stub_process_wait(exitstatus)
         call_exec_windows('test command', {}, @stdin, @stdout, @stderr)
       end
 
+      context 'cwd option' do
+        let(:cwd) { 'cwd' }
+        it "should execute the command in the specified working directory" do
+          Dir.expects(:chdir).with(cwd)
+          Process.expects(:create).with(
```

Comment:
though we might need to check for `Dir.expects` we don't also need to make an expectation for `Process.create`, this should be a stub and we can remove the `.with` altogether.

Reply:
Yup, I can change this to `stubs`